ta_services/keymaster/entry.c

```diff
@@ -44,7 +60,12 @@ TEE_Result TA_InvokeCommandEntryPoint(void __unused *sess, uint32_t cmd,
 				      uint32_t pt,
 				      TEE_Param params[TEE_NUM_PARAMS])
 {
+	if (!is_configured)
```

Comment:
Will always be `true`?

I'd rather that the `km_` functions that care if we're configured or not checks this themselves.

Reply:
Why will always be `true`?

Similar to comment below, do you mean for all other functions in `km.c` to call this function rather than just calling it once in `entry.c`?

Reply:
It will always be true as you're just taking the address of the function instead of calling it.

Yes, all top level `km_` functions that cares should call this.

Reply:
Oops sorry.. careless. Will fix.

Reply:
> Yes, all top level km_ functions that cares should call this.

Ok. Please remember to check `km_is_configured()` first for every km_ function you (including @igoropaniuk and @jbech-linaro included) implement. I'll add this check to `km_add_rng_entropy()` that was just merged, so it can be used as a reference.